src/main/scala/mesosphere/marathon/upgrade/AppStopActor.scala

```diff
@@ -15,14 +15,21 @@ class AppStopActor(
     val taskTracker: TaskTracker,
     val eventBus: EventStream,
     app: AppDefinition,
+    val config: UpgradeConfig,
     val promise: Promise[Unit]) extends StoppingBehavior {
 
   override var idsToKill: mutable.Set[Task.Id] = taskTracker.appTasksLaunchedSync(app.id).map(_.taskId).to[mutable.Set]
 
   def appId: PathId = app.id
```

Comment:
I know you didn't add this line, but could you also add `override` to the method declaration?